### ISSUE TITLE
Do we want to enforce some variant of pylint's style (and set up a CI hook)

### ISSUE THREAD

**Issue by thetazero** [PyCubed-Mini/software_example_beepsat#3]
Could be good for our code quality especially since none of us are seasoned python programmers.

**Comment by jbwillis** [comment]
Yes, this would be fantastic. I think the standard is to use pep8 for python coding style.

**Comment by thetazero** [comment]
Do you think we should use this library, it seems the pep8 CLI tool was renamed to pycodestyle: https://pypi.org/project/pycodestyle/

**Comment by thetazero** [comment]
Added some basic .vscode config file to automatically use pep8 for python coding style (using pycodestyle) in #2. 
Also configured it to set max line length to 120, to make it less annoying.

**Comment by thetazero** [comment]
Implemented in #11
